src/mapeo-manager.js

```diff
@@ -219,10 +219,16 @@ export class MapeoManager {
     // 5. Write project name and any other relevant metadata to project instance
     await project.$setProjectSettings(settings)
 
+    // 6. Write device info into project
+    const deviceInfo = await this.getDeviceInfo()
+    if (deviceInfo.name) {
+      await project.$setOwnDeviceInfo({ name: deviceInfo.name })
+    }
```

Comment:
wondering if this is desired behavior or not. keeping this means that one should call `manager.setDeviceInfo()` before adding or creating a project. Otherwise, attempting to use the `$member` namespace causes issues.